docs/rfcs/1015-auto-enrollment-device.md

```diff
@@ -0,0 +1,135 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Auto-Enrollment Device
```

Comment:
This RFC is in line with what we've discussed so far :+1:. I would only change the name as the "enrollment" term has already been banned from Parsec. Also, the auto

This "remote device" was previously referred as "Device Registration Key" in the sense that it was "a key/device used to register new devices". If that name does not fit, maybe "Registration Device" or simply "Account Vault Device" (since there will only be one per user account/organization).

Reply:
RFC has been renamed "Registration Device"